include/call.hpp

```diff
@@ -322,7 +322,7 @@ class call : virtual public task, virtual public listener, public virtual socket
     BIO *m_bio;
 #endif
 
-    int _callDebug(const char *fmt, ...);
+    int _callDebug(const char *fmt, ...) __attribute__((format(printf, 2, 0)));
```

Comment:
Yeah, in case you're curious:

~~~
./call.hpp:325:57: error: format attribute cannot specify the implicit this argument as the format string
    int _callDebug(const char *fmt, ...) __attribute__((format(printf, 1, 0)));


~~~